### PR TITLE
add install build and channel params for jenkinsfile and additional  jenkinsfile for  batch test

### PR DESCRIPTION
**Pull request by aganesh-suse**
#### Proposed Changes ####

Inorder to do the one click jenkins job, the individual jobs need to take installation build and channel params from the jenkins job, instead of from the tfvars file/text content. To accomplish this, the change needs to happen in 2 places: 
1. Jenkinsfile needs to update the TFVARS file content with params provided from the jenkins job. (This change)
2. Jenkins Jobs, need to have the vars VERSION_OR_COMMIT, CHANNEL, INSTALL_MODE as params for all jobs. They will no longer be from the TFVARS file/text content. This change will be made in the jenkins job builder files.  

#### Testing ####

Checklist:
1. If your PR changes anything on or related to Jenkins, run it pointing to your branch to make sure it's okay.

3. Verify code lint; we should not have errors.

4. Update the documentation if needed.

5. Update makefile and docker run if adding new tests.

6. Run your tests at least 4 times with all configurations needed and possible.

7. If needed test with different os types.

### CODE REVIEW

**Review by mdrahman-suse — APPROVED** [review]
LGTM

**Review by VestigeJ — APPROVED** [review]

File: scripts/Jenkinsfile_batch_test [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+          ])
+        }
+        dir("./") {
+          stage('Run Parallel Jobs') {
+            echo "Running ${testdir.size()} tests: ${testdir}"
+            def jobs = [:]
+
+            for (int i = 0; i < testdir.size(); i++) {
+              def testname = testdir[i]
+              def actual_testdir_name = testname
```
Comment (fmoral2) [inline scripts/Jenkinsfile_batch_test:50]: lets name consistently, or camel or snake

File: scripts/Jenkinsfile_batch_test [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+            ],
+            userRemoteConfigs: repo
+          ])
+        }
+        dir("./") {
+          stage('Run Parallel Jobs') {
+            echo "Running ${testdir.size()} tests: ${testdir}"
+            def jobs = [:]
+
+            for (int i = 0; i < testdir.size(); i++) {
```
[the remark this answers is not in the document]
Comment (aganesh-suse) [reply scripts/Jenkinsfile_batch_test]: fixed

File: scripts/Jenkinsfile_batch_test (hunk 1 above)
Comment (aganesh-suse) [reply scripts/Jenkinsfile_batch_test]: fixed

**Review by fmoral2 — DISMISSED** [review]

File: scripts/Jenkinsfile_batch_test [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+              ]
+              echo "Params are: ${params}"
+
+              echo "Trigger build job for: ${test_name}"
+              switch(test_name) {
+                case "validatecluster":
+                  jobs[test_name] = { build job: "${env.PRODUCT_NAME}_validate_cluster", parameters: params }                 
+                case "restartservice":
+                  jobs[test_name] = { build job: "${env.PRODUCT_NAME}_restart_service", parameters: params }                  
+                case "dualstack":
```
Comment (mdrahman-suse) [inline scripts/Jenkinsfile_batch_test:107]: Do we want to add dual stack and deploy rancher as part of this batch run? @rancher-max

Comment (aganesh-suse) [reply scripts/Jenkinsfile_batch_test:107]: it can be here. we dont have to pass it in the TEST_DIRECTORY_LIST if we dont want to?

Comment (rancher-max) [reply scripts/Jenkinsfile_batch_test:107]: yes we do want to add those please!

**Review by mdrahman-suse — APPROVED** [review]
LGTM

**Review by fmoral2 — APPROVED** [review]

**Review by rancher-max — APPROVED** [review]

**Pull request merged**